packages/react-storage/src/components/StorageBrowser/views/Controls/Controls.tsx

```diff
@@ -27,6 +28,7 @@ export interface Controls {
   Navigate: typeof NavigateControl;
   Refresh: typeof RefreshControl;
   Search: typeof SearchControl;
+  Checkbox: typeof CheckboxControl;
```

Comment:
We don't want to add anymore of these `Controls`, @cshfang is working towards removing them all

Reply:
moved